src/snapred/backend/dao/state/PixelGroupingParameters.py

```diff
@@ -10,9 +10,15 @@ class PixelGroupingParameters(BaseModel):
     # True => this group is fully masked
     isMasked: bool
 
-    # the average two-theta (radians) for this pixel grouping
+    # the mean sample to pixel distance (metres) for this pixel grouping
+    L2: float
+
+    # the mean two-theta (two times the polar angle in radians) for this pixel grouping
     twoTheta: float = Field(ge=0.0, le=2.0 * np.pi)
 
+    # the mean azimuth (radians) for this pixel grouping
+    azimuth: float = Field(ge=-np.pi, le=np.pi)
```

Comment:
Why is the range being selected this way, instead of $[0, 2\pi)$?  What would happen to a value like $3\pi$?  Can a modulo operator be slipped in here?

Reply:
The problem is that `DetectorInfo::azimuthal` ends up using `std::atan2` without modification.  I would have preferred `[0, 2π)` myself for the azimuth, as I believe that it's more standard.  Note that this also means that there should be no need for a _modulo_ operator here.